docs/verify/stabilizer-circuit-simulation.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
<p>I think the intro for this page could be fleshed out a bit more, and I think this is also a page where we should think carefully about the narrative we're trying to put forward about simulation. Atm this reads like "now we can simulate 50 qubits yay!" but I think we should be clearer about the limitations and why doing this kind of simulation helps you to "verify" what you're doing before running on real hardware. The tone should be more focused on using this as a stepping stone before moving on to real devices</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/322/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Also I think some extra info on some of the terms introduced in the first paragraph would be helpful. E.g. I'd recommend either explain the definition of a "stabilizer" or provide a link to a definition elsewhere

Reply:
oh hang on, I realised you do define them later on. OK in that case I change my feedback lol. Instead could you move the definition higher up, or hint somehow in the intro that you're going to explain what they are as well as how to use them